modules/rn_3_7_0.adoc

```diff
@@ -0,0 +1,67 @@
+[[rn-3-700]]
+== Version 3.7.0
+
+=== quay / clair / quay-builder
+
+Added/Changed:
+
+* With {productname} 3.7, users have the ability to report storage consumption and to contain registry growth by establishing configured storage quota limits. Although there are no limitations on a Quay organization's storage consumption, on-premise Quay users are now equipped with quota reporting and quota management capabilities to manage the capacity limits of their environment. (link:https://issues.redhat.com/browse/PROJQUAY-302[PROJQUAY-302], link:https://issues.redhat.com/browse/PROJQUAY-253[PROJQUAY-253])
+
+* The bare-metal constraint required to run builds has been removed by adding an additional build option which does not contain the virtual machine layer. As a result, builds can be run on virtualized platforms. Backwards compatibility to run previous build configurations are also available. (link:https://issues.redhat.com/browse/PROJQUAY-295[PROJQUAY-295])
+
+* {productname} can now act as a proxy cache to circumvent pull-rate limitations from upstream registries. This feature also accelerates pull performance, because images are pulled from the cache rather than upstream dependencies. Cached images are only updated when the upstream image digest differs from the cached image, reducing rate limitations and potential throttling. (link:https://issues.redhat.com/browse/PROJQUAY-465[PROJQUAY-465])
```

Comment:
> circumvent pull-rate limitations

nit: circumvent makes me think that we're giving users a way to surpass the rate limit...which is true for pulls of the same image but not if they try to pull an excessive number of different images. Thinking maybe `mitigate` is a better word that `circumvent`?